guis/common/db_classes/straw_location.py

```diff
@@ -375,43 +378,48 @@ def forceAddStraw(self, straw, position, commit=True):
             )
             target_position = StrawPosition(location=self.id, position=position)
             target_position.commit()
-
-        # Get positions where straw is currently located.
-        # Nothing stopping from a straw existing in two places at once, so get
-        # all possible. This is a list of StrawPresent objects.
-        straw_presents = (
-            self._queryStrawPresents().filter(StrawPresent.straw == straw).all()
-        )
-        if len(straw_presents) > 1:
-            logger.debug(f"Straw ST{straw} located in > 1 position :(.")
-
-        # Remove straw from all current positions, unless one of them is the
-        # target position, in which case quit when done.
-        done = False
-        for straw_present in straw_presents:
-            if straw_present.getStrawPosition() == target_position:
-                logger.debug(
-                    f"Straw ST{straw} already in target location {target_position}."
-                )
-                done = True
-            else:
-                straw_present.remove()
-        if done:
-            return
-
-        # Clear the target position of all straws
-        target_position.unloadPresentStraws()
-
-        # Add the straw to the target position (add entry to straw_position
-        # table).
-        straw_present = StrawPresent(
-            straw=straw, position=target_position.id, present=true()
-        )
+        if past is False:
```

Comment:
`if not past`

Reply:
But also, `past` isn't descriptive enough for me. I think I prefer `historical`.